onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -289,6 +289,9 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformers(
                                                                                onnxruntime::kJsExecutionProvider};
       const InlinedHashSet<std::string_view> cpu_dml_eps = {onnxruntime::kCpuExecutionProvider,
                                                             onnxruntime::kDmlExecutionProvider};
+      const int64_t qdq_matmulnbits_accuracy_level =
+          std::stoi(session_options.config_options.GetConfigOrDefault(kOrtSessionOptionsQDQMatMulNBitsAccuracyLevel,
```

Comment:
consider using ParseStringWithClassicLocale

https://github.com/microsoft/onnxruntime/blob/beb2496748b112ba0b2525c14f1093acbd98c7aa/include/onnxruntime/core/common/parse_string.h#L79

it attempts to parse the whole string instead of allowing leading whitespace or trailing unparsed characters

Reply:
fix in next iter